VmAgent.Core/Interfaces/ProcessRunner.cs

```diff
@@ -92,7 +92,11 @@ public Task CollectLogs(string id, string logsFolder, ISessionHostManager sessio
 
         private (string, string) GetExecutableAndArguments(SessionHostsStartInfo sessionHostsStartInfo, int instanceNumber)
         {
-            string[] parts = sessionHostsStartInfo.StartGameCommand.Split(' ', StringSplitOptions.RemoveEmptyEntries);
+            var command = sessionHostsStartInfo.StartGameCommand.Replace("\\ ", "\\SPACE");
```

Comment:
C# generally accepts / in Windows paths, and Linux-based systems naturally want / as well. I'd say a conversion of "\\" -> "/" would be more universal and safe. Additionally, when I think of what would typically replace a space in a path, I think of "_" or sometimes "-". I don't really think of "SPACE".